src/character.cpp

```diff
@@ -132,9 +132,20 @@ bool Character::move_effects()
             remove_effect("in_pit");
         }
     }
+    if (has_effect("grabbed")){
+        if ( get_dex() > get_str() ? dice(get_dex(), 7) : dice(get_str(), 7) < get_effect_dur("downed") ){
```

Comment:
I'd like it to do something along those lines, but I'm not sure where that implementation would be. Idealy, I'd think you could either attack an adjacent enemy, or attempt to escape. I'm not sure how to make that happen though. Can I check to see if the player is attacking from mon_move?